tests/core/test_dataset.py

```diff
@@ -314,11 +315,13 @@ def test_find_missing_datasets() -> None:
 # External databases require credentials passed through environment variables
 SNOWFLAKE_URL_TEMPLATE = "snowflake://FIDESCTL:{}@ZOA73785/FIDESCTL_TEST"
 SNOWFLAKE_URL = SNOWFLAKE_URL_TEMPLATE.format(
-    os.getenv("SNOWFLAKE_FIDESCTL_PASSWORD", "")
+    quote_plus(os.getenv("SNOWFLAKE_FIDESCTL_PASSWORD", ""))
```

Comment:
@ThomasLaPiana this resolves the failing external pytest error, will search through issues now and either add a note or create a new one